.github/workflows/build.yml

```diff
@@ -33,6 +33,8 @@ jobs:
         run: |
           ver=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout -f rta)
           if [[ $ver == *"SNAPSHOT"* ]]; then
+              export DISPLAY=:90
+              Xvfb -ac :90 -screen 0 1280x1024x24 > /dev/null 2>&1 &
               mvn -B -ntp deploy -f rta
```

Comment:
Since we are already running tests in the previous step, there shouldn't be a need to run them again.

Reply:
makes sense, fixed